compiler/src/iree/compiler/Codegen/LLVMGPU/test/ROCDL/config_vector_distribute_gfx940.mlir

```diff
@@ -1,6 +1,6 @@
 // RUN: iree-opt --split-input-file --iree-gpu-test-target=gfx940 --iree-codegen-llvmgpu-use-vector-distribution \
 // RUN:   --iree-codegen-llvmgpu-use-unaligned-gemm-vector-distribution --iree-codegen-llvmgpu-use-igemm=false \
-// RUN:   --pass-pipeline="builtin.module(iree-llvmgpu-select-lowering-strategy)" %s | FileCheck %s
+// RUN:   --pass-pipeline="builtin.module(func.func(iree-linalg-ext-convert-attention-to-online-attention), iree-llvmgpu-select-lowering-strategy)" %s | FileCheck %s
```

Comment:
Can we update the tests instead? I'm not a fan of this. This test file should just test select-lowering-strategy